x-pack/plugins/endpoint/public/applications/endpoint/services/ingest.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { HttpFetchOptions, HttpStart } from 'kibana/public';
+import { GetDatasourcesRequest } from '../../../../../ingest_manager/common/types/rest_spec';
+import { PolicyData } from '../types';
+
+const INGEST_API_ROOT = `/api/ingest_manager`;
+const INGEST_API_DATASOURCES = `${INGEST_API_ROOT}/datasources`;
+
+interface GetDatasourcesResponse {
+  items: PolicyData[];
+  total: number;
+  page: number;
+  perPage: number;
+  success: boolean;
+}
+
+interface GetDatasourceResponse {
+  item: PolicyData;
+  success: boolean;
+}
+
+/**
+ * Retrieves a list of endpoint specific datasources (those created with a package.name of
+ * `endpoint`) from Ingest
+ * @param http
+ * @param options
+ */
+export const sendGetEndpoingDatasources = (
```

Comment:
style suggestion: `Endpoint` is misspelled. Also, I'm not clear on the name of this function. Based on the doc comment, I would think  `endpointSpecificDatasource` might be a good candidate for the name. Also, for clarity, would you consider adding an explicit return type?
I think `Promise GetDatasourcesResponse>` would be correct

Reply:
Sounds good. I'll change the name to `fetchEndpointSpecificDatasources()`. 
FYI - These are in fact Endpoint datasources - we will define them in the endpoint package and when creating a new one, the `package.name` will be set to `endpoint` and tie back to the package from where it was defined.